app/views/wiki/history.html.erb

```diff
@@ -26,9 +26,15 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 See COPYRIGHT and LICENSE files for more details.
 
 ++#%>
-<%= toolbar title: @page.title %>
 <% html_title t("activerecord.models.wiki"), t(:label_history) %>
-<% breadcrumb_paths(*(@page.ancestors.reverse.collect { |parent| link_to h(parent.breadcrumb_title), { id: parent, project_id: parent.project } } + [h(@page.breadcrumb_title)])) %>
+<%=
+  render Primer::OpenProject::PageHeader.new do |header|
+    header.with_title { h(@page.title) }
```

Comment:
This feels weird.. I'd expect something like "History of {title}" and get rid of that second headline. Or just "History" and represent the title in the breadcrumb